app/src/test/java/com/example/habitsmasher/ui/dashboard/HabitListTest.java

```diff
@@ -34,30 +35,32 @@ public void addHabit_validHabitAddition_expectHabitAddedToList(){
     @Test
     public void editHabit_validEdit_expectHabitToBeEdited() {
         long habitId = 0;
-        Habit habit = new Habit("Title 1", "Reason 1", new Date(), habitId, EMPTY_HABIT_EVENTS_LIST);
+        Habit habit = new Habit("Title 1", "Reason 1", new Date(), "MO", habitId, EMPTY_HABIT_EVENTS_LIST);
         _habitList.addHabitLocal(habit);
         Date newDate = new Date();
         int habitToEdit = 0;
         String newTitle = "Title 2";
         String newReason = "Reason 2";
-        _habitList.editHabitLocal(newTitle, newReason, newDate, habitToEdit);
+        DaysTracker tracker = new DaysTracker("MO WE FR");
+        _habitList.editHabitLocal(newTitle, newReason, newDate, tracker, habitToEdit);
         Habit editedHabit = _habitList.getHabitList().get(habitToEdit);
 
         assertEquals(newTitle, editedHabit.getTitle());
         assertEquals(newReason, editedHabit.getReason());
         assertEquals(newDate, editedHabit.getDate());
         assertEquals(habitId, editedHabit.getId());
+        assertEquals(tracker.getDays(),editedHabit.getDays());
     }
 
     @Test
     public void deleteHabitLocally_withValidHabitPosition_expectHabitDeletedFromList() {
         Date today = new Date();
         ArrayList<Habit> localHabitList = _habitList.getHabitList();
 
-        Habit habitToDelete = new Habit("Habit 2", "Reason 2", today, 1, EMPTY_HABIT_EVENTS_LIST);
-        _habitList.addHabitLocal(new Habit("Habit 1", "Reason 1", today, 0, EMPTY_HABIT_EVENTS_LIST));
+        Habit habitToDelete = new Habit("Habit 2", "Reason 2", today, "MO WE FR", 1, EMPTY_HABIT_EVENTS_LIST);
```

Comment:
Since the days of the week aren't really **relevant** to this test, and considering that this string is used in multiple places, let's extract this to a constant `DAYS_OF_THE_WEEK`.

Reply:
Done. 👍